src/test/java/uk/gov/hmcts/cmc/claimstore/services/ReferenceNumberServiceTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.cmc.claimstore.repositories.ReferenceNumberRepository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ReferenceNumberServiceTest {
+
+    private static String CITIZEN_REFERENCE = "000MC001";
+    private static String LEGAL_REFERENCE = "000LR001";
+
+    private ReferenceNumberService referenceNumberService;
+
+    @Mock
+    private ReferenceNumberRepository referenceNumberRepository;
+
+    @Before
+    public void setup() {
+        referenceNumberService = new ReferenceNumberService(referenceNumberRepository);
+    }
+
+    @Test
+    public void shouldGetReferenceNumberForCitizen() {
+
+        when(referenceNumberRepository.getReferenceNoForCitizen()).thenReturn(CITIZEN_REFERENCE);
+
+        Boolean claimRepresented = false;
+        String referenceNumber = referenceNumberService.getReferenceNumber(claimRepresented);
+
+        //then
```

Comment:
If you are using BDD-style comments then please use the full structure, i.e.
```
// given 
...
// when
...
// then
...
```
or remove them completely. Consider [`BDDMockito`](http://static.javadoc.io/org.mockito/mockito-core/2.15.0/org/mockito/BDDMockito.html) as well.